cmd/odf/restore/crds.go

```diff
@@ -20,7 +21,16 @@ var deletedCmd = &cobra.Command{
 	},
 	Run: func(cmd *cobra.Command, args []string) {
 		k8sutil.SetDeploymentScale(cmd.Context(), root.ClientSets.Kube, root.OperatorNamespace, "ocs-operator", 0)
-		pkgrestore.RestoreCrd(cmd.Context(), root.ClientSets, root.OperatorNamespace, root.StorageClusterNamespace, args)
+		var groupName string
+		var versionResource string
+		if args[0] == "storageclusters" {
```

Comment:
Is there a check for length of `args` before this? If not, this could crash.

Reply:
The command already includes argument validation through Cobra with the line:
```
Args: cobra.RangeArgs(1, 2),
```

This ensures that if the number of arguments is outside the 1–2 range, Cobra will return an error and prevent further execution. As shown in the output, passing three arguments results in:

```
$ ./bin/odf restore deleted  a b c
Error: accepts between 1 and 2 arg(s), received 3
```